tests/cudnn_fusion_test.py

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
+os.environ["XLA_FLAGS"] = "--xla_gpu_cublas_fallback=false"
```

Comment:
Unfortunately you cannot set flags like this globally because we run multiple test in the same process.

You can pass them as an option to `.compile()` if you use the AOT compilation path.

Any chance we can just write a test robust to this?

Reply:
Changed to compile({"xla_gpu_cublas_fallback": False}). I think this is the most robust way: we do expect XLA to choose the fastest backend for GEMM fusions by default. Otherwise I could change the shapes such that the fused cased would be N times as fast as the unfused one, but that would make the test slower and rely on performance measurements. We could also make the test ignore which GEMM fusion backend does XLA use in the end, but that could allow some failures.